packages/business-rules/src/schemas/householder-appeal/insert.js

```diff
@@ -15,10 +15,19 @@ const insert = pinsYup
     decisionDate: pinsYup.date().transform(parseDateString).nullable(),
     submissionDate: pinsYup.date().transform(parseDateString).nullable(),
     state: pinsYup.string().oneOf(Object.values(APPEAL_STATE)).default(APPEAL_STATE.DRAFT),
-    appealType: pinsYup.string().oneOf(Object.values(APPEAL_ID)),
+    appealType: pinsYup
+      .string()
+      .oneOf([...Object.values(APPEAL_ID), null])
+      .nullable(),
```

Comment:
I think it would be better to make the null check conditional here as in other rules.

Reply:
Updated